LayoutTests/platform/mac/TestExpectations

```diff
@@ -2265,10 +2265,6 @@ webkit.org/b/231924 inspector/css/modify-css-property.html [ Pass Failure ]
 
 webkit.org/b/237172 imported/w3c/web-platform-tests/speech-api/SpeechSynthesis-speak-twice.html [ Pass Failure Crash ]
 
-# Note: These tests were previously marked as flaky failures for Big Sur Release WK2 for webkit.org/b/235681, this may need to be re-added once webkit.org/b/248067 is resolved
-webkit.org/b/248067 imported/w3c/web-platform-tests/html/canvas/element/manual/imagebitmap/createImageBitmap-drawImage.html [ Pass Failure ]
-webkit.org/b/248067 imported/w3c/web-platform-tests/html/canvas/element/manual/imagebitmap/createImageBitmap-flipY.html [ Pass Failure ]
-
```

Comment:
I do not understand how this change is fixing the flakiness in these tests especially createImageBitmap-flipY.html. Have they been already stable even without this patch? Or should webkit.org/b/248067 be resolved a dupe this one?

Reply:
It's probably not, they just seemed to consistently pass (and I was changing the expectation for them anyway), so figured I'd try to enable.